dependencies/buster/msgpack/ruby-tests.rake

```diff
@@ -0,0 +1,5 @@
+require 'gem2deb/rake/spectask'
+
+Gem2Deb::Rake::RSpecTask.new do |spec|
+  spec.pattern = './spec/**/*_spec.rb'
```

Comment:
this needs to catch `./spec/*_spec.rb` and `./spec/cruby/*_spec.rb` but not `./spec/jruby/*_spec.rb`

Reply:
Done